sdk/communication/communication-phone-numbers/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Remove this file.

Reply:
same question here

Reply:
We're still shipping the license with the package, right? Are there work items that licenses need to be removed for all services?